src/pykeen/models/base.py

```diff
@@ -476,129 +474,6 @@ def predict(
 
         raise ValueError(f"Unknown target={target}")
 
-    def get_all_prediction_df(
-        self,
-        *,
-        k: Optional[int] = None,
-        batch_size: int = 1,
-        **kwargs,
-    ) -> Union[ScorePack, pd.DataFrame]:
-        """Compute scores for all triples, optionally returning only the k highest scoring.
-
-        .. note:: This operation is computationally very expensive for reasonably-sized knowledge graphs.
-        .. warning:: Setting k=None may lead to huge memory requirements.
-
-        :param k:
-            The number of triples to return. Set to None, to keep all.
-        :param batch_size:
-            The batch size to use for calculating scores.
-        :param kwargs: Additional kwargs to pass to :func:`pykeen.models.predict.get_all_prediction_df`.
-        :return: shape: (k, 3)
-            A tensor containing the k highest scoring triples, or all possible triples if k=None.
-        """
-        from .predict import get_all_prediction_df
-
-        warnings.warn("Use pykeen.models.predict.get_all_prediction_df", DeprecationWarning)
-        return get_all_prediction_df(model=self, k=k, batch_size=batch_size, **kwargs)
-
-    def get_prediction_df(
-        self,
-        *args,
-        **kwargs,
-    ) -> Union[ScorePack, pd.DataFrame]:
-        """Get predictions for the head, relation, and/or tail combination.
-
-        :param args: Positional arguments to pass to :func:`pykeen.models.predict.get_prediction_df`.
-        :param kwargs: Keyword arguments to pass to :func:`pykeen.models.predict.get_prediction_df`.
-        :return: shape: (k, 3)
-            A dataframe with columns based on the settings or a tensor. Contains either the k highest scoring triples,
-            or all possible triples if k is None
-        """
-        from .predict import get_prediction_df
-
-        warnings.warn("Use pykeen.models.predict.get_prediction_df", DeprecationWarning)
-        return get_prediction_df(self, *args, **kwargs)
-
-    def get_head_prediction_df(
```

Comment:
I'm hesitant to see all of these high level wrappers go away

Reply:
oh wait inside the model it's fine

Reply:
they also have been deprecated for some time now :slightly_smiling_face: